src/helpers/issue.ts

```diff
@@ -133,6 +133,31 @@ export const getAllIssueComments = async (issue_number: number): Promise<Comment
   return result;
 };
 
+export const issueAutopayCheck = async (issue_number: number) => {
+  let comments = await getAllIssueComments(issue_number);
+  let lastAutopayStatus = null;
+
+  const regex = /Autopay is (disabled|enabled)/;
```

Comment:
This seems brittle. Perhaps we should have a more robust system for bot comments as a "database" or at least we should share the string between the read and write function so that we will not have a situation where a developer only updates one of the two, breaking this system.

Reply:
@pavlovcik i did not fully get your point above, are you saying we should store the autopay states to a DB instead?